packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/terminateEmail/billing-autoRenew-terminateEmail.html

```diff
@@ -5,19 +5,10 @@
     data-terminate-service="$ctrl.terminateEmail()"
     data-go-back="$ctrl.goBack()"
 >
-    <div data-ng-if="!$ctrl.isHosting">
-        <oui-message data-type="warning">
-            <span
-                data-translate="autorenew_service_EMAIL_DOMAIN_terminate_info"
-            ></span>
-        </oui-message>
-        <p
-            data-translate="autorenew_service_EMAIL_DOMAIN_terminate_procedure"
-        ></p>
-    </div>
-    <oui-message data-ng-if="$ctrl.isHosting" data-type="warning">
+    <oui-message data-type="warning" class="mb-4">
```

Comment:
```suggestion
    <oui-message type="warning" class="mb-4">
```
If you want 🎉

Reply:
Not mandatory but could be done in one place for all occurrences.

Thanks for the suggestion.